repos.yaml

```
@@ -1615,3 +1615,45 @@ repositories:
     maintainers:
       - name: John Teague
         email: john@fasterbytes.com
+  - name: prometheus-community
+    url: https://prometheus-community.github.io/helm-charts
+    maintainers:
+      - email: gianrubio@gmail.com
+        name: gianrubio
+      - email: zanhsieh@gmail.com
+        name: zanhsieh
+      - email: miroslav.hadzhiev@gmail.com
+        name: Xtigyro
+      - email: mattias.gees@jetstack.io
+        name: mattiasgees
+      - email: hfernandez@mesosphere.com
+        name: hectorj2f
+      - email: cedric@desaintmartin.fr
+        name: desaintmartin
+      - email: me@sota.sh
+        name: rsotnychenko
+      - email: mail@torstenwalter.de
+        name: torstenwalter
+      - email: asher@asherfoa.com
+        name: asherf
+      - email: 8890118+timm088@users.noreply.github.com
+        name: timm088
+      - email: github.gkarthiks@gmail.com
+        name: gkarthiks
+      - email: ssheehy@firescope.com
+        name: steven-sheehy
+      - email: juan.chimienti@gmail.com
+        name: juanchimienti
+      - email: monotek23@gmail.com
+        name: monotek
+      - email: okgolove@markeloff.net
+        name: okgolove
+      - email: carlos@carlosbecker.com
+        name: caarlos0
+      - name: vsliouniaev
+      - email: christian.staude@staffbase.com
+        name: cstaud
+      - email: maxime@root314.com
+        name: miouge1
+      - email: arcadie.condrat@gmail.com
+        name: acondrat
```

Thanks for this PR 🙂

This field should be maintainers of the repo (who make repo-level changes as needed), not the chart maintainers. I was just asking myself who the repo maintainers should be, and I realized I don't know the answer 🤔 

I just made this issue for a transparent community-led process: https://github.com/prometheus-community/helm-charts/issues/38

If you would like to list this repo in the Hub sooner, you may temporarily add my name since I'm currently on he admin team:

```suggestion
      - email: scott@r6by.com
        name: scottrigby
```

It can always be updated later with the admin team info once other members join. Also, since Helm Hub will be migrating to Artifact Hub soon this is probably fine #439  The repo is already listed there, and we will add repo maintainers to the prometheus Org there so there's not a single point of failure.

---

I was having that question in my mind and decided to add all the current chart maintainers. 🤣 

I guess as you suggested, I will just add you there. It makes sense with your reasonings you have added in the comment.